<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginBox</class>
 <widget class="QDialog" name="loginBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QIPSClient Login</string>
  </property>
  <widget class="QLineEdit" name="userBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>351</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>351</width>
     <height>28</height>
    </rect>
   </property>
   <property name="inputMask">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="userLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>User Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
